src/pages/slide-editor/[slideName].tsx

```
@@ -0,0 +1,584 @@
+
+  // スライド移動関数
+  const nextSlide = useCallback(() => {
+    slideStore.setState((state) => ({
+      currentSlide: Math.min(state.currentSlide + 1, slideCount - 1),
+    }))
+  }, [slideCount])
+
+  const prevSlide = useCallback(() => {
+    slideStore.setState((state) => ({
+      currentSlide: Math.max(state.currentSlide - 1, 0),
+    }))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)